dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSP23JSP22ServerTest.java

```diff
@@ -124,7 +124,7 @@ public void testJsp23to22FeatureChange() throws Exception {
      * @throws Exception
      */
     @Test
-    @SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
+    @SkipForRepeat({SkipForRepeat.NO_MODIFICATION, SkipForRepeat.EE10_FEATURES})
```

Comment:
@volosied can you open an issue so we add an EE10 pages-3.1 test for this functionality? If you can provide a link here that would be good so I can ensure we track it. Thanks!

Reply:
This test should already be added under the new 3.1 FAT.

Reply:
Oops. It's not in this PR because I never pulled in in from my other branch. I'll add it in soon.

Reply:
Test added